MyString.java

```diff
@@ -49,8 +75,17 @@ public static boolean subsetOf(String str1, String str2) {
      * @return a string consisting of the characters of str, separated by spaces.
      */
     public static String spacedString(String str) {
-        //// Replace the following statement with your code
-        return null;
+        int length= str.length();
+        String newString="";
+        if (str.length()==0) {
+            return ""+"";
+        }

```

Comment:
"" is neutral value in concat. 
This is equivalent to write `return 0 + 0` in some addition function